internal/service/odb/cloud_vm_cluster.go

```diff
@@ -6,6 +6,7 @@ package odb
 import (
 	"context"
 	"errors"
+	"regexp"
```

Comment:
we should replace the standard `regex` library with `"github.com/YakDriver/regexache"`. This is used throughout the provider and provides better caching